src/apollo/client.ts

```diff
@@ -0,0 +1,53 @@
+import { ApolloClient, InMemoryCache, createHttpLink } from "@apollo/client";
+import { setContext } from "@apollo/client/link/context";
+import { onError } from "@apollo/client/link/error";
+import { headersForRequest } from "../Components/ResponseFieldWithApollo";
+
+const httpLink = createHttpLink({
+  uri: "https://rickandmortyapi.com/graphql",
+});
+
+type MessErr = {
+  graphErr: string;
+  netErr: string;
+};
+
+export const messError: MessErr = {
+  graphErr: "",
+  netErr: "",
+};
+
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors) {
+    graphQLErrors.forEach(({ message, locations }) => {
+      locations
+        ? (messError.graphErr =
+            `[Error GraphQL]: Message: ${message}` +
+            "\n" +
+            `Location: Line: ${locations[0].line} Column: ${locations[0].column}`)
+        : (messError.graphErr = `[Error GraphQL]: Message: ${message} `);
+    });
+  } else messError.graphErr = ``;
```

Comment:
you should use curly braces in `else` statement
```suggestion
  } else {
    messError.graphErr = ``;
  }
```
https://eslint.org/docs/latest/rules/curly#all